src/rest_framework_dso/filters.py

```diff
@@ -394,7 +394,16 @@ def get_ordering(self, request, queryset, view):
             # Can adjust 'self' as this instance is recreated each request.
             if "sorteer" in request.query_params:
                 self.ordering_param = "sorteer"
-        return super().get_ordering(request, queryset, view)
+
+        ordering = super().get_ordering(request, queryset, view)
+        if ordering is None:
+            return ordering
+
+        # convert to snake_case, preserving `-` if needed
+        correct_ordering = [
+            "-".join([to_snake_case(y) for y in x.split("-")]) for x in ordering
+        ]
+        return correct_ordering
```

Comment:
to_snake_case() zou eerder in een bovenliggende class moeten gebeuren (DynamicOrderingFilter?).

Oorspronkelijk was het idee om deze restframework_dso package ooit extern te maken. Misschien moeten we het anders maar wel samenvoegen?

Reply:
Dat was ik ook aan het denken, maar:
 - RangeFilter doet hetzelfde hier
 - er is geen DynamicOrderingFIlter en er moet veel meer aangepasst om deze aan te makken.

Dus denk ik het is wat te veel voor de snelle fix :)

Reply:
eens :-)